keyboards/dz60/keymaps/_bonfire/keymap-parts/global-functions.c

```diff
@@ -0,0 +1,14 @@
+/**
+ * Key Mods Function
+ *
+ * This turns 3d, 4b, 4d and 4e into arrow keys when tapped,
+ * and RSFT, RCTL, RGUI and HYPR when held.
+ *
+ * @DEPRECATED -- view: https://docs.qmk.fm/#/custom_quantum_functions
+ */
+const uint16_t PROGMEM fn_actions[] = {
+    [0] = ACTION_MODS_TAP_KEY(MOD_RSFT, KC_UP),
+    [1] = ACTION_MODS_TAP_KEY(MOD_RCTL, KC_DOWN),
+    [2] = ACTION_MODS_TAP_KEY(MOD_RGUI, KC_LEFT),
+    [3] = ACTION_MODS_TAP_KEY(MOD_HYPR, KC_RIGHT),
```

Comment:
These are QMK-native keycodes, as:

- `MT(MOD_RSFT, KC_UP)`
- `MT(MOD_RCTL, KC_DOWN)`
- `MT(MOD_RGUI, KC_LEFT)`
- `MT(MOD_HYPR, KC_RIGHT)`

respectively, which can be used directly in your `layers.c` Lines 39-40 (via an alias you define, if you wish).

If you make that change, you don't need this file at all.